### PR TITLE
Added functionality to add post without uploading an image

### PR DESCRIPTION
**Pull request by partik03**
This pr is in response to the issue #25 by @mathiasayivor

### CODE REVIEW

**Comment by narayan954** [comment]
Can you please resolve the conflicts @partik03

**Comment by partik03** [comment]
@narayan954 I have resolved the conflicts kindly merge it

**Review by narayan954 — CHANGES_REQUESTED** [review]
please check my comment on the code

File: src/components/ImgUpload.jsx [hunk 1]
```diff
@@ -14,44 +14,55 @@ function ImgUpload(props) {
       setImage(e.target.files[0]);
     }
   };
-  const handleUpload = () => {
-    setUploadingPost(true);
```
[the remark this answers is not in the document]
Comment (narayan954) [reply src/components/ImgUpload.jsx:18]: alright I am doing it myself :) , so it won't be delayed

**Review by narayan954 — APPROVED** [review]
looks good :)

**Pull request merged**